expcore/Mini_games.lua

```diff
@@ -207,8 +263,39 @@ function Mini_games.error_in_game(error_game)
     Mini_games.stop_game()
     game.print("an error has occured things may be broken, error: "..error_game)
 end
+
+local kick_all =
+function()
+    for i,player in ipairs(game.connected_players) do
+        game.kick_player(player,"You cant stay here")
+    end
+end
+
+Commands.new_command('kill_all','Command to stop a mini_game.')
+:register(function(_,_)
+    kick_all()
+end)
+
+Commands.new_command('stop_games','Command to stop a mini_game.')
```

Comment:
This command doesnt actully stop the minigame, this just sends everyone to the lobby.

Reply:
again it  it copy paste as no one can use this anyway (not added to config.role.lua